mobile/src/main/java/org/openhab/habdroid/util/PrefExtensions.kt

```diff
@@ -90,6 +91,14 @@ fun SharedPreferences.shouldRequestHighResChart(): Boolean {
     return getBoolean(PrefKeys.CHART_HQ, true)
 }
 
+fun SharedPreferences.getImageWidgetScalingType(): WidgetImageView.ImageScalingType {
+    return if (getBoolean(PrefKeys.IMAGE_WIDGET_SCALE_TO_FIT, true)) {
```

Comment:
This `if` statement should be inversed, right?

Reply:
Good catch, fixed.